wesnoth-nr.po

```diff
@@ -8711,8 +8410,8 @@ msgid ""
 "Lords of Light — or Darkness — guide you on your path. For those of you who "
 "are staying — come, we have much to do."
 msgstr ""
-"Ještě jednou vám všem děkuji. Pro ty, kteří odcházejí — kéž Bohové Světla — "
-"nebo Temnot — vedou vaše kroky. Pro ty, kdo zůstávají — máme před sebou "
+"Ještě jednou vám všem děkuji. Pro ty, kteří odcházejí, kéž Bohové Světla – "
```

Comment:
```suggestion
"Ještě jednou vám všem děkuji. Pro ty, kteří odcházejí, kéž bohové Světla – "
```

Reply:
Ale bohové mohou být Bohové. I když uznávám, že jsem to asi všude neupravil stejně.